src/core/Bot.py

```diff
@@ -76,6 +76,9 @@ def __init__(self, **kwargs):
         self.lockdown_msg: str = None
         self._BotBase__cogs = commands.core._CaseInsensitiveDict()
 
+        self.message_cache: Dict[int, Any] = {}
+        # TODO: Use LRU caching
```

Comment:
Only thing scare me is, the `self.message_cache` will keep growing and growing....

Reply:
Rn, I don't think it'll grow enough to cause any impact on the performance of the application itself, I expect its length to be less than 40k  with the current approach (in around 7 days).

I guess implementing TTL cache can be better than this but I am not sure if we can serialize `discord.Message()`. Maybe we can but I am not really sure.

Reply:
I hope you have enough RAM. TTL, MRU, LRU take lot of RAM